### ISSUE TITLE
Build problem: building project with CLI generates JSON parsing error.

### ISSUE THREAD

**Issue by espen42** [enonic/starter-react4xp#18]
Creating a new project using the enonic CLI, following the instructions from https://developer.enonic.com/templates/react4xp#create_project - no setup changed.

Environment: Windows 10, french system language.

**Comment by amineamami** [comment]
Full-stack trace:

```
 Deploying to sandbox 'com.phx.demo'...

> Task :npmInstall
npm WARN rollback Rolling back readable-stream@2.3.6 failed (this is probably harmless): EPERM: operation not permitted, lstat 'C:\Users\amine.amami\WebstormProjects\enonic\reactxp\node_modules\fsevents\node_modules'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 724 packages from 265 contributors and audited 11952 packages in 53.625s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

> Task :config_react4xp FAILED
undefined:1
{buildRuntimeClient:false,outputFileName:C:\\Users\\amine.amami\\WebstormProjects\\enonic\\reactxp/build/react4xp_constants.json,buildEnv:development,BUILD_ENV:development,masterConfigFileName:build/react4xp_constants.json,overwriteConstantsFile:true,buildExternals:tr
ue,verbose:false}
 ^

SyntaxError: Unexpected token b in JSON at position 1
    at JSON.parse (<anonymous>)
    at buildConstants (C:\Users\amine.amami\WebstormProjects\enonic\reactxp\node_modules\react4xp-buildconstants\index.js:25:26)
    at Object.<anonymous> (C:\Users\amine.amami\WebstormProjects\enonic\reactxp\node_modules\react4xp-buildconstants\cli.js:15:16)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':config_react4xp'.
> Process 'command 'node'' finished with non-zero exit value 1

* Try:
* Get more help at https://help.gradle.org

BUILD FAILED in 56s
3 actionable tasks: 2 executed, 1 up-to-date

```

**Comment by espen42** [comment]
Environment: Windows 10, french system language.
Creating a new project using the enonic CLI, following the instructions from https://developer.enonic.com/templates/react4xp#create_project - no setup changed.

**Comment by espen42** [comment]
Fixed: starter version 0.2.10.
